src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -1179,7 +1179,7 @@ C2V_VMENTRY_0(jboolean, setCountersSize, (JNIEnv* env, jobject, jint new_size))
   return JavaThread::resize_all_jvmci_counters(new_size);
 C2V_END
 
-C2V_VMENTRY_0(jint, allocateCompileId, (JNIEnv* env, jobject, ARGUMENT_PAIR(method), int entry_bci))
+C2V_VMENTRY_0(juint, allocateCompileId, (JNIEnv* env, jobject, ARGUMENT_PAIR(method), int entry_bci))
```

Comment:
This is called from Java code which does not have an unsigned int. I think we need to leave this as `jint` here.